workbench-core/example/infrastructure/src/__tests__/__snapshots__/example-stack.test.ts.snap

```diff
@@ -15,11 +15,11 @@ Object {
         },
         Object {
           "id": "AwsSolutions-APIG4",
-          "reason": "@aws/workbench-core-authentication implemented",
+          "reason": "@aws/workbench-core-authorization implemented at app level",
```

Comment:
@maghirardelli looks like some changes were not merged correctly. This file does not exist in `develop` anymore

Reply:
Yeah, looking at this diff: https://github.com/aws-solutions/solution-spark-on-aws/compare/develop...merge-develop-to-proj-apis-11-14, it seems I need to delete:
- workbench-core/example/infrastructure/integration-tests/tests/datasets/create.test.ts
- workbench-core/example/infrastructure/src/__tests__/__snapshots__/example-stack.test.ts.snap
- workbench-core/example/infrastructure/src/__tests__/example-stack.test.ts

Additionally, it looks like NOTICE in infrastructure package is changed which I assume is a format on save error. Let me try to remedy these. Thanks for pointing it out!